samples/tutorial/sym1/symIO.c

```diff
@@ -0,0 +1,170 @@
+// --------------------------------------------------------------------------
```

Comment:
We don't want any new programs in the tutorial. Please remove it or move it to `targettest/sym1`.

Reply:
**FIXED**  Please check and confirm.